cmd/agent/app/reporter/grpc/reporter.go

```diff
@@ -71,3 +74,24 @@ func (r *Reporter) send(spans []*model.Span, process *model.Process) error {
 	}
 	return err
 }
+
+// addTags appends jaeger tags for the agent to every span it sends to the collector.
+func addTags(spans []*model.Span, agentTags []model.KeyValue) []*model.Span {
+	for _, span := range spans {
+		span.Tags = append(span.Tags, agentTags...)
```

Comment:
why span.Tags instead of span.Process.tags?  I think a typical use case here would be adding some metadata that's constant for all spans from a given application